functions/slack/SlackKnowledgeGraphSearch/Function.cs

```diff
@@ -0,0 +1,175 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Kgsearch.v1;
+using Google.Apis.Kgsearch.v1.Data;
+using Google.Apis.Services;
+using Google.Cloud.Functions.Framework;
+using Google.Cloud.Functions.Invoker;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+[assembly: FunctionsStartup(typeof(SlackKnowledgeGraphSearch.Startup))]
```

Comment:
And this is probably important to include as well? For the Startup class it's cleaner to have a separate file.

Reply:
Aha... if I put the Startup class *and* the attribute in the same file, that can have the relevant `using` directives. I've done that in an extra commit - see what you think.